libcnb/CHANGELOG.md

```diff
@@ -2,10 +2,14 @@
 
 ## [Unreleased]
 
+## [0.7.0] 2022-04-08
+
 - Allow compilation of libcnb.rs buildpacks on Windows. Please note that this does not imply Windows container support, it's meant to allow running unit tests without cross-compiling. ([#368](https://github.com/heroku/libcnb.rs/pull/368))
 - Expose `runtime::libcnb_runtime_detect`, `runtime::libcnb_runtime_build` and their related types for advanced use-cases. Buildpack authors should not use these. ([#375](https://github.com/heroku/libcnb.rs/pull/375)).
 - Only create layer `env`, `env.build` and `env.launch` directories when environment variables are being set within them ([#385](https://github.com/heroku/libcnb.rs/pull/385)).
 - Add `WriteLayerError::MissingExecDFile` error to ease debugging when an exec.d path is missing ([#387](https://github.com/heroku/libcnb.rs/pull/387)).
+- Update project URLs for the GitHub repository move to the `heroku` org ([#388](https://github.com/heroku/libcnb.rs/pull/388)).
+- Update `libcnb-data` (which provides the types in the `data` module) from `0.5.0` to `0.6.0` - see the [libcnb-data changelog](../libcnb-data/CHANGELOG.md#060-2022-04-08) ([#391](https://github.com/heroku/libcnb.rs/pull/391)).
```

Comment:
I deliberately only mentioned `libcnb-data` (and not the other monorepo crate cross-dependencies), since it's the only major release that's depended on by another libcnb* crate, and so (a) the others will get pulled in just by updating lockfiles, (b) there isn't anything much of note to mention for the others anyway.